proto/v1beta1/vulnerability.proto

```diff
@@ -146,7 +146,14 @@ message Vulnerability {
   // security tracker.
   google.protobuf.Timestamp source_update_time = 6;
 
-  // Next free ID is 7.
+  // This field tracks the CVE record for this vulnerability
+  string cve = 7;
```

Comment:
We already record this at the top-level, when we attach the Occurrence to the note and record the CVE number in `note_name`: https://github.com/grafeas/grafeas/blob/master/proto/v1beta1/grafeas.proto#L214. Does this help?

Reply:
The only concern as expressed below is having the same resource url across multiple occurences with different note name. (each for CVE & CWEs)

Reply:
This would be WAI: we create multiple `VulnerabilityOccurrence`s for a given image, each of which is attached to a different `VulnerabilityNote` that represents a CVE. Does this help clarify the model?